release/_patcher.sh

```diff
@@ -25,9 +25,7 @@ function report_jenkins_url {
 		echo "}"
 	) > "${LIFERAY_RELEASE_HOTFIX_BUILD_ID}"
 
-	rsync -Dlprtvz --chown=501:501 --no-perms "${_BUILD_DIR}"/patcher-status/ test-3-1::patcher/
-
-	ssh test-3-1 "chown -R 501:501 /mnt/mfs-hdd1-172.16.168/patcher"
+	rsync -Dlprtvz --chown=501:501 --no-perms "${_BUILD_DIR}"/patcher-status/ /mnt/patcher-shared/patcher/
```

Comment:
@laszlobereczki, can you please give more details about this change? 

As fair as I remember, the code above was splited in 2 different commands to fix a Patcher Portal outage, please see https://github.com/liferay/liferay-docker/commit/d11414498261986fecbc64d38f9e6c4a41f35023 and https://liferay.atlassian.net/browse/LRP-4877.